public/locales/en-US/translations.json

```diff
@@ -452,5 +452,9 @@
   "language_ja-JP": "Japanese",
   "xchain_account_claim_count": "XChain Account Claim Count",
   "xchain_account_create_count": "XChain Account Create Count",
-  "min_signer_quorum": "Minimum weight <0>{{quorum}}</0> required"
+  "min_signer_quorum": "Minimum weight <0>{{quorum}}</0> required",
+  "holder": "Holder",
+  "claws_back": " Claws back",
+  "claws_back_from": "<source/> claws back from <destination/>",
+  "instruct_to_claw": "It was instructed to claw back at most "
```

Comment:
```suggestion
  "instruct_to_claw": "The max clawback amount is ____"
```

Reply:
This should be a translation that takes a param, because other languages might have this in a different order